deployments/operator/crd/bases/deviceplugin.intel.com_qatdeviceplugins.yaml

```diff
@@ -90,6 +90,15 @@ spec:
                 description: NodeSelector provides a simple way to constrain device
                   plugin pods to nodes with particular labels.
                 type: object
+              preferredAllocationPolicy:
+                description: PreferredAllocationPolicy sets the mode of allocating
+                  GPU devices on a node. See documentation for detailed description
+                  of the policies. Only valid when SharedDevNum > 1 is set.
```

Comment:
you'll have to run `make generate` to get this comment fixed.

Reply:
Ah, why is it still like this, sorry. I already did with `make generate`.. but idk why it's still like that. Thanks.!